### PR TITLE
[IMP] Time Off: added multiple request section

### PR DESCRIPTION
**Pull request by larm-odoo**
Was requested ot make this change in the following tasks:

https://www.odoo.com/odoo/project.task/4376969
https://www.odoo.com/odoo/unassigned-tasks/4380705

The 'Mode' filed is gone an instead, you must use the 'Multiple Requests' option instead, to allocate to more than one employee at a time.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @jero-odoo - this is ready for a peer review when you have a moment!

File: content/applications/hr/time_off/allocations.rst [hunk 1]
```diff
@@ -74,28 +76,60 @@ information:
...
+  the :guilabel:`Employee Tag` field. When a tag is selected, *all* employees with that tag receive
+  the allocation.
+
+Next, select the :guilabel:`Time Off Type` using the drop-down menu. Once a :guilabel:`Time Off
+Type` is selected, the placeholder name, 'Allocation Request', changes to the name of the selected
+:guilabel:`Time Off Type`, including the amount of days. Change the name of the allocation, if
+desired.
+
+Fill out the remainder of the :ref:`Multiple Requests form <time_off/allocation-form>`, then click
+:guilabel:`Create Allocaitons` when done.
```
Comment (jero-odoo) [inline content/applications/hr/time_off/allocations.rst:127]:
```suggestion
:guilabel:`Create Allocations` when done.
```

File: content/applications/hr/time_off/allocations.rst [hunk 2]
```diff
@@ -74,28 +76,60 @@ information:
...
+When allocating time off, it is common to allocate time to several employees at once. This is done
+using the :guilabel:`Multiple Requests` feature.
+
+To allocate time to multiple employees in a single allocation, navigate to :menuselection:`Time Off
+app --> Management --> Allocations`. Then, click the :icon:`fa-gear` :guilabel:`(gear)` icon in the
+upper-left corner, then click :icon:`fa-users` :guilabel:`Multiple Requests`. This reveals a
+:guilabel:`Multiple Requests` pop-up window.
+
+This form is identical to the :guilabel:`Allocation` form, with one additional field:
+:guilabel:`Mode`. The :guilabel:`Mode` field determines how multiple employees are selected.
```
Comment (jero-odoo) [inline content/applications/hr/time_off/allocations.rst:98]:
```suggestion
This form is identical to the :guilabel:`Allocation` form, with an additional :guilabel:`Mode` field. The :guilabel:`Mode` field determines how multiple employees are selected.
```

**Review by jero-odoo — APPROVED** [review]
Hey @larm-odoo everything looks good, just a few notes. Also, something to note, it looks like the multiple requests image was not compressed (nor was the image titled "new-allocation"). I would double check this before moving it to final. Thanks!

File: content/applications/hr/time_off/allocations.rst (hunk 1 above)
Comment (larm-odoo) [reply content/applications/hr/time_off/allocations.rst:127]: Ugh- this wasn't underlined (so I must not have my spell-checker activated or something). I'll get that fixed- thank!

**Comment by larm-odoo** [comment]
Hi @samueljlieber - this is ready for a tech review!

File: content/applications/hr/time_off/allocations.rst [hunk 3]
```diff
@@ -19,7 +19,9 @@ Allocations`.
 
 This presents a list of all current allocations, including their respective statuses.
 
-Click :guilabel:`New` to allocate time off, and a blank allocation form appears.
+Click :guilabel:`New` to allocate time off, and a blank :guilabel:`Allocation` form appears.
+
+.. _time_off/allocation-form:
```
Comment (samueljlieber) [inline content/applications/hr/time_off/allocations.rst:24]: Please move this custom anchor above the "Allocate time off" heading :)
```suggestion
```

File: content/applications/hr/time_off/allocations.rst [hunk 4]
```diff
@@ -74,28 +76,60 @@ information:
...
 
+Multiple Allocations
+--------------------
+
+When allocating time off, it is common to allocate time to several employees at once. This is done
+using the :guilabel:`Multiple Requests` feature.
+
+To allocate time to multiple employees in a single allocation, navigate to :menuselection:`Time Off
+app --> Management --> Allocations`. Then, click the :icon:`fa-gear` :guilabel:`(gear)` icon in the
+upper-left corner, then click :icon:`fa-users` :guilabel:`Multiple Requests`. This reveals a
```
Comment (samueljlieber) [inline content/applications/hr/time_off/allocations.rst:94]: Moving forward, we will refer to this as the "Actions" icon, when using the Actions menu in the UI
```suggestion
app --> Management --> Allocations`. Then, click the :icon:`fa-gear` :guilabel:`(Actions)` icon in
the upper-left corner, then click :icon:`fa-users` :guilabel:`Multiple Requests`. This reveals a
```

File: content/applications/hr/time_off/allocations.rst [hunk 5]
```diff
@@ -74,28 +76,60 @@ information:
...
+
+This form is identical to the :guilabel:`Allocation` form, with an additional :guilabel:`Mode`
+field. The :guilabel:`Mode` field determines how multiple employees are selected.
+
+Using the drop-down menu, select one of the following :guilabel:`Modes`:
+
+- :guilabel:`By Employee`: This option allows for the selection of multiple individual employees
+  that are unrelated in terms of department, company, or tags. Selecting this reveals an
+  :guilabel:`Employees` field. Select the employees to receive the allocation in the
+  :guilabel:`Employees` field. There is n limit to the amount of employees that can be selected.
```
Comment (samueljlieber) [inline content/applications/hr/time_off/allocations.rst:105]: Typo: I think this should be "no"?
```suggestion
  :guilabel:`Employees` field. There is no limit to the amount of employees that can be selected.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo! Nice work on this PR. Your changes look good to me, approving with a couple quick fixes, thank you!
...
@robodoo delegate=larm-odoo

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**